app/controllers/badges_controller.rb

```diff
@@ -1,6 +1,7 @@
 class BadgesController < ApplicationController
   before_action :authenticate_user!
   before_action :set_badge, only: [:show]
+  load_and_authorize_resource except: [:require, :redeem]
```

Comment:
@frmendes I'm already using the `is_organizer` for that. The problem here is that if a user does not have `can :manage, Badge` in its abilities they cannot access the redeem badge page...